XenModel/XenAPI-Extensions/VM.cs

```diff
@@ -683,7 +683,14 @@ void VirtualizationTimer_Elapsed(object sender, ElapsedEventArgs e)
         private Timer VirtualizationTimer = null;
 
         [Flags]
-        public enum VirtualisationStatus { UNKNOWN = 0, OPTIMIZED = 1, PV_DRIVERS_NOT_INSTALLED = 2, PV_DRIVERS_OUT_OF_DATE = 4 };
+        public enum VirtualisationStatus
+        {
+            PV_DRIVERS_NOT_INSTALLED    = 0,

```

Comment:
Yes, I agree with this - removed confusing PV_DRIVERS_NOT_INSTALLED from the enum.